Source/Core/VideoCommon/TextureCacheBase.h

```diff
@@ -68,6 +69,11 @@ class TextureCacheBase
 		// Keep an iterator to the entry in textures_by_hash, so it does not need to be searched when removing the cache entry
 		std::multimap<u64, TCacheEntryBase*>::iterator textures_by_hash_iter;
 
+		// This is used to keep track of both:
+		//   * efb copies used by this partially updated texture
+		//   * partially updated textures which refer to this efb copy
+		std::unordered_set<TCacheEntryBase*> refrences;
```

Comment:
It gets cleared in freeTexture()

Unless there is some other code path which can result in releasing a texture entry???